showcase/src/styles/styles.scss

```diff
@@ -1,9 +1,13 @@
 @import "basscss-missing-variables.pcss"; /* FIXME: Remove this import once the issue in basscss is solved (see: https://github.com/basscss/basscss/issues/257) */
 @import "~basscss/css/basscss.css";
 @import "~prismjs/themes/prism-okaidia.css";
-
+/*
+Import the local theme file first to set the correct variables, see:
```

Comment:
Shouldn't this comment also be included in the starter?

Reply:
I moved it to `styles/theme.scss` for starter and showcase. I think it is more logical.